libs/messaging/src/main/kotlin/uk/gov/justice/digital/hmpps/messaging/NotificationHandler.kt

```diff
@@ -1,12 +1,18 @@
 package uk.gov.justice.digital.hmpps.messaging
 
+import com.fasterxml.jackson.databind.JsonMappingException
 import uk.gov.justice.digital.hmpps.converter.NotificationConverter
 import uk.gov.justice.digital.hmpps.message.Notification
+import uk.gov.justice.digital.hmpps.telemetry.TelemetryService
 
 interface NotificationHandler<T : Any> {
     val converter: NotificationConverter<T>
     fun handle(notification: Notification<T>)
     fun handle(message: String) {
-        handle(converter.fromMessage(message))
+        try {
+            handle(converter.fromMessage(message))
+        } catch (e: JsonMappingException) {
```

Comment:
Does this apply to all services?  Probably makes sense to do this only for risk-assessment-scores-to-delius if possible?